dapr/actor/runtime/config.py

```diff
@@ -106,4 +188,11 @@ def as_dict(self) -> Dict[str, Any]:
             configDict.update(
                 {'remindersStoragePartitions': self._reminders_storage_partitions})
 
+        configDict['entitiesConfig'] = []
+        for entityConfig in self._entitiesConfig:
+            configDict['entitiesConfig'].append(entityConfig.as_dict())
+            entities.add(entityConfig._actor_type)
+
+        configDict['entities'] = list(entities)
```

Comment:
Why add a new variable and then convert to list? 
Can't it be 
`configDict['entities'] = list(self._entities)` ?

Reply:
we need to keep a method local set to dedupe in a method local context, but at the end we must return a list.